routers/api/v1/user/app.go

```diff
@@ -74,3 +75,39 @@ func CreateAccessToken(ctx *context.APIContext, form api.CreateAccessTokenOption
 		Sha1: t.Sha1,
 	})
 }
+
+// DeleteAccessToken delete access tokens
+func DeleteAccessToken(ctx *context.APIContext) {
+	// swagger:operation DELETE /users/{username}/tokens/{token} user userCreateToken
+	// ---
+	// summary: Create an access token
+	// consumes:
+	// - application/json
+	// produces:
+	// - application/json
+	// parameters:
+	// - name: username
+	//   in: path
+	//   description: username of user
+	//   type: string
+	//   required: true
+	// - name: token
+	//   in: path
+	//   description: token to be deleted
+	//   type: string
+	//   required: true
+	// responses:
+	//   "204":
+	//     "$ref": "#/responses/empty"
+	tokenID := ctx.ParamsInt64(":id")
```

Comment:
Question for reviewers: This deletes based on ID (int), however we only return the name (string) of the token above. So this works, but users need to know what the ID (int) is before using this. My question is: Should this be changed to delete based on token name (string)?

Reply:
How does GitHub API works?

Reply:
@lafriks Their docs say they use the integer ID of the token: https://developer.github.com/v3/oauth_authorizations/#delete-an-authorization

They also list the ID in the response when listing the tokens, so I will keep this PR the same, and I'll update this PR to include the ID in the List endpoint.